lib/libzutil/zutil_import.c

```diff
@@ -1374,7 +1374,7 @@ label_offset(uint64_t size, int l)
  * labels found will be returned in num_labels when non-NULL.
  */
```

Comment:
Add comment:
`* The pool GUID is saved into guid if guid is non-NULL.`

Reply:
Added.